src/pages/industries/software.jsx

```diff
@@ -0,0 +1,219 @@
+  ],
+  withImage: true,
+  imageSrc: SoftwareImage1,
+  ImageAlt: 'cilium saas tools and dbs illustration',
+};
+
+const bulletSection2 = {
+  heading:
+    "Streamline Onboarding New Applications and Teams and Accelerate debugging with Cilium's Hubble Service Map",
+  paragraphs: [
+    "Cilium's Hubble Service Map feature allows platform teams to furnish application teams with a portal to view their own workloads, dependencies, and network flows. This self-service model makes it easier to identify service connections, calls, and even DNS operations, all crucial for diagnosing problems. The service map helps streamline the onboarding of new applications and teams and accelerates the debugging process.",
+    'By mitigating disputes between application and infrastructure operation teams, Cilium accelerates the investigation of application-layer issues, enabling frictionless, self-service root cause analysis for application teams.',
+  ],
+  withImage: true,
+  imageSrc: SoftwareImage2,
+  ImageAlt: 'hubble 5xx graphs',
+};
+
+const bulletSection3 = {
+  heading: 'Build Secure Kubernetes Multi-tenant Environments With Cilium ',
+  paragraphs: [
+    "For enterprises building muti-tenant architectures, Cilium guarantees isolation between tenants and ensures different teams can safely run on the same platform. With Cilium's multi-tenant observability features, teams can easily access application metrics without compromising security. Traditional logging systems often fall short when segregating data by tenants. Cilium provides multi-tenant, self-service access using the OpenID Connect (OIDC) standard. Teams can have RBAC-based access to relevant data and connectivity metrics, such as network policy drops, DNS lookup failures, and more.",
+  ],
+  withImage: true,
+  imageSrc: SoftwareImage3,
+  ImageAlt: 'hubble architecture',
+};
+
+const softwareTalks = [
+  {
+    title: 'Designing and Securing a Multi-Tenant Runtime Environment at the New York Times ',
+    description:
+      'The New York Times team was building multi-tenant Kubernetes clusters to complement the Internal Developer platform and needed networking that could securely scale up to 100+ Nodes',
+    videoSrc: 'https://www.youtube.com/embed/9FDpMNvPrCw',
+  },
+
+  {
+    title: 'The Past, Present and Future of Cilium and Hubble at Palantir',
+    description:
+      'Vlad Ungureanu from Palantir details how they use Cilium and they plan to use it in future',
+    videoSrc: 'https://www.youtube.com/embed/3K5WJ_h5PhI',
+  },
+  {
+    title: 'Global Gaming Infrastructure with Cilium',
+    description:
+      'How Wildlife Studios built a Global Multi Cluster Gaming Infrastructure with Cilium',
+    videoSrc: 'https://www.youtube.com/embed/_1t3bXzptP0',
+  },
+];
+
```

Comment:
replace with talks from datadog and cosmonic. Clickhouse will be coming out at Kubecon